### DISCUSSION TITLE
How to compile the libvips with cgif library

### DISCUSSION THREAD

**Discussion by gonuit** [libvips/libvips#3176] [Q&A]
Hi everyone 👋🏻 
I'm trying to compile the libvips from the source with the cgif library. What should I do to make the cgif visible?
I have cgif built from source and installed under the path `/usr/lib/aarch64-linux-gnu/libcgif.so` (the platform is aarch64-linux, and cgif has version 0.3.0).

I tried specifying the env variables `PKG_CONFIG_PATH`, `LD_LIBRARY_PATH` and `LIBRARY_PATH` and setting the `-Dcgif=enabled` option in the `meson setup` command. However, I still getting:

```
GIF save with cgif                : NO
```

Is there something I'm missing? What should I do to build it correctly 🤔

**Comment by vikasjio** [comment]
You also need to install libimagequant and libimagequant-devel. Vips uses these for image color quantization for GIF/PNG writing.
